test/n_security_token_registry.js

```diff
@@ -993,7 +993,8 @@ contract("SecurityTokenRegistry", accounts => {
 
         it("Should get the tickers by owner", async () => {
             let tickersList = await I_STRProxied.getTickersByOwner.call(token_owner);
-            assert.equal(tickersList.length, 4);
+            console.log(tickersList);
+            assert.equal(tickersList.length, 3);
```

Comment:
```suggestion
            assert.equal(tickersList.length, 4);
```

Reply:
hmm in local test it was reverting, because 4th ticker was null because of that bug so it should be 3 tickers. let me run it again to confirm.

Reply:
on local if I run without the fix then it give me this 
 ['0x4445540000000000000000000000000000000000000000000000000000000000',
  '0x4554480000000000000000000000000000000000000000000000000000000000',
  '0x504f4c5900000000000000000000000000000000000000000000000000000000',
  '0x0000000000000000000000000000000000000000000000000000000000000000' ]
so 4th is null that should not be added in the output. so I run the fixed code then I always return the 3 ticker 
 ['0x4445540000000000000000000000000000000000000000000000000000000000',
  '0x4554480000000000000000000000000000000000000000000000000000000000',
  '0x504f4c5900000000000000000000000000000000000000000000000000000000' ]
I don't understand why it is not reflecting on the CI. 
Note - I run individual test file not the full test suite

Reply:
Contracts are reset by truffle on every test file so running individually should not matter.
In the test code, I can see 4 tickers being registered. (lines 387,409,549,575)